model/pull_request.go

```diff
@@ -28,9 +27,37 @@ type PullRequest struct {
 	BuildLink           string
 	URL                 string
 	CreatedAt           time.Time
-	Merged              sql.NullBool
+	Merged              *bool
 	MergeCommitSHA      string `db:"-"`
-	MaintainerCanModify sql.NullBool
-	MilestoneNumber     sql.NullInt64
-	MilestoneTitle      sql.NullString
+	MaintainerCanModify *bool
+	MilestoneNumber     *int64
+	MilestoneTitle      *string
+}
+
+func (pr *PullRequest) GetMerged() bool {
```

Comment:
Not a blocking thing but, I'd prefer Go convention:
```suggestion
func (pr *PullRequest) Merged() bool {
```

Reply:
@isacikgoz Wouldn't that conflict with the field name?

Reply:
Ahh yeah .. that's true, the field is exported. Well, we could unexport and have it accessible only via a getter. But it's out of scope for this PR.

Reply:
I'll happily do a separate PR addressing it if I get a green light.

Reply:
I don't think that's necessary, just for the sake of sticking to a convention. A comment explaining why the "Get" part is needed should suffice.